src/api/che.ts

```diff
@@ -437,10 +437,24 @@ export class CheHelper {
       () => { })
   }
 
+  async getWorkspaces(cheURL: string, accessToken?: string): Promise<any[]> {
+    const all: any[] = []
+    const maxItems = 30
+    let skipCount = 0
+
+    do {
```

Comment:
Is it make sense do cycle if default maxItems is 30? https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java#L344